operators/multiclusterobservability/controllers/placementrule/placementrule_controller.go

```diff
@@ -408,29 +392,35 @@ func deleteManagedClusterRes(c client.Client, namespace string) error {
 	return nil
 }
 
+func updateManagedClusterList(obj client.Object) {
+	vendor, ok := obj.GetLabels()["vendor"]
+	if !ok && vendor != "OpenShift" {
+		return
+	}
+	obs, ok := obj.GetLabels()["observability"]
+	if ok && obs == "disabled" {
+		return
+	}
+	managedClusterList = util.RemoveDuplicates(append(managedClusterList, obj.GetName()))
+}
```

Comment:
I'm thinking can we filter the `managedCluster` resources by the labels in filter cache?

Reply:
Done